src/windows/GeolocationProxy.js

```diff
@@ -181,10 +178,11 @@ module.exports = {
             callbacks = ids[clientId];
```

Comment:
Also, it might be worth it to either add an assertion or throw an exception here if the passed `clienId` is not in `locs` or `ids`.